coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -186,13 +227,38 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
         metadata = cls.Metadata(*(str(docstyle_settings.get(req_setting, ''))
                                   for req_setting in metadata_settings))
 
+        class_padding = (cls.ClassPadding('', '') if
+                         docstyle_settings.get('class_padding', '').value == ''
+                         else cls.ClassPadding(
+                         *(int(padding) for padding in tuple(
+                             docstyle_settings.get('class_padding', ''))))
+                         )
```

Comment:
These constructs are super hard to read. Would look much nicer as a try/except block